src/open_inwoner/scss/components/Map/_Map.scss

```diff
@@ -20,3 +20,15 @@
 .p + .map {
   margin-top: var(--spacing-large);
 }
+
+.leaflet-content-name {
+  & > h4 {
+    margin: 0 !important;
+  }
+}
+
+.leaflet-content-details {
+  display: flex;
+  row-gap: var(--spacing-small);
+  flex-wrap: wrap;
```

Comment:
You would need to add this to it: 
`flex-direction: column;`
Everything else looks good! 
Explanation: The leaflet-map address etc. will look weird if locations have long names + while still breaking neatly on a hyphen (the location details would be **next** to each other in stead of on top of each other, making it difficult to read without a margin.):
![image](https://user-images.githubusercontent.com/118177951/220887731-62afbd5d-41ef-43fb-bcc1-a6a58304d5b6.png)